cmd/kueuectl/app/list/list_workload_test.go

```diff
@@ -0,0 +1,470 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package list
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/cli-runtime/pkg/genericiooptions"
+	fakediscovery "k8s.io/client-go/discovery/fake"
+	kubetesting "k8s.io/client-go/testing"
+
+	kueue "sigs.k8s.io/kueue/apis/kueue/v1beta1"
+	"sigs.k8s.io/kueue/apis/visibility/v1alpha1"
+	"sigs.k8s.io/kueue/client-go/clientset/versioned/fake"
+	cmdtesting "sigs.k8s.io/kueue/cmd/kueuectl/app/testing"
+	utiltesting "sigs.k8s.io/kueue/pkg/util/testing"
+)
+
+func TestWorkloadCmd(t *testing.T) {
+	testStartTime := time.Now()
```

Comment:
Could we use the fake time instead of real time?

Reply:
Do you mean this one `func NewFakeClock(t time.Time)`?

Reply:
I meant that couldn't we introduce the same approach as controllers like this:

- https://github.com/kubernetes-sigs/kueue/blob/4b9744b29bec33bbd34e01a66f09002b478a6493/pkg/controller/core/workload_controller.go#L117
- https://github.com/kubernetes-sigs/kueue/blob/4b9744b29bec33bbd34e01a66f09002b478a6493/pkg/controller/core/workload_controller_test.go#L807

But I'm ok with follow-up.

Reply:
I meant that couldn't we introduce the same approach as controllers like this:

- https://github.com/kubernetes-sigs/kueue/blob/4b9744b29bec33bbd34e01a66f09002b478a6493/pkg/controller/core/workload_controller.go#L117
- https://github.com/kubernetes-sigs/kueue/blob/4b9744b29bec33bbd34e01a66f09002b478a6493/pkg/controller/core/workload_controller_test.go#L807

But I'm ok with follow-up.

Reply:
I propose to make it in another PR.

Reply:
#2287